### PR TITLE
Remove log propagation from child to root

### PR DESCRIPTION
**Pull request by ramo-j**
This removes duplicate log entries when some modules are used:

Before:
```
[2022-03-07 03:37:45,074] [dftimewolf          ] INFO     Running module: LocalFilesystemCopy
[2022-03-07 03:37:45,074] [LocalFilesystemCopy ] INFO     <path> -> /tmp/dftimewolf_local_fs0t8tq4ta
2022-03-07 03:37:45 [INFO] <path> -> /tmp/dftimewolf_local_fs0t8tq4ta
[2022-03-07 03:37:45,079] [LocalFilesystemCopy ] SUCCESS  <path> was compressed into /tmp/dftimewolf_local_fs0t8tq4ta/<output>.tgz
2022-03-07 03:37:45 [SUCCESS] <path> was compressed into /tmp/dftimewolf_local_fs0t8tq4ta/<output>.tgz
[2022-03-07 03:37:45,079] [dftimewolf          ] INFO     Module LocalFilesystemCopy finished execution

```

After:
```
[2022-03-07 03:42:41,743] [dftimewolf          ] INFO     Running module: LocalFilesystemCopy
[2022-03-07 03:42:41,743] [LocalFilesystemCopy ] INFO     <path> -> /tmp/dftimewolf_local_fs76lmm6m2
[2022-03-07 03:42:41,748] [LocalFilesystemCopy ] SUCCESS  <path> was compressed into /tmp/dftimewolf_local_fs76lmm6m2/<output>.tgz
[2022-03-07 03:42:41,748] [dftimewolf          ] INFO     Module LocalFilesystemCopy finished execution
```

### CODE REVIEW

**Review by itsmvd — APPROVED** [review]

File: dftimewolf/lib/collectors/grr_hosts.py [hunk 1]
```diff
@@ -264,8 +264,8 @@ def _AwaitFlow(self, client: Client, flow_id: str) -> None:
       try:
         status = client.Flow(flow_id).Get().data
       except grr_errors.UnknownError:
-        msg = 'Unable to stat flow {0:s} for host {1:s}'.format(
-            flow_id, client.data.os_info.fqdn.lower())
+        msg = (f'Unable to start flow {flow_id} for host '
```
Comment (tomchop) [inline dftimewolf/lib/collectors/grr_hosts.py:267]: Drive-by comment - `stat` was not a typo here, we're trying to get info on the flow, not start it. "stat" comes from the error you get from the GRR API when the flow doesn't exist (similar to filesystem `stat`), IIRC.
